ts-eks/test/pulumi/index.ts

```diff
@@ -0,0 +1,45 @@
+import * as aws from "@pulumi/aws";
+import * as awsx from "@pulumi/awsx";
+import { PolicyPack, validateResourceOfType, StackValidationPolicy } from "@pulumi/policy";
+
+const lambdaPolicy: StackValidationPolicy = {
+    name: "lambda-test",
+    description: "Lambda integration tests.",
+    enforcementLevel: "mandatory",
+    validateStack: async (args, reportViolation) => {
+        const lambdas = args.resources.filter(r => r.isType(aws.lambda.Function));
```

Comment:
This is checking for lambdas which we don't have any, also I think the memory values are off

Reply:
yeah im going to delete pulumi policies for now, just so we can get something in